test/functional/cypress/integration/ecs/clusters_list.spec.js

```diff
@@ -65,22 +65,25 @@ describe('Amazon ECS: aws-prod-ecsdemo cluster', () => {
       .get('a[title="f8757e00-184d-4288-b535-4124a739e7be"]')
       .click();
 
-
     cy.get('.details-panel > .header')
       .get('instance-details-header')
       .get('.header-text')
       .get('h3:contains("f8757e00-184d-4288-b535-4124a739e7be")');
 
-    cy.get('.details-panel > .content > .collapsible-section').eq(0)
+    cy.get('[data-test-id="instanceDetails.content"]')
+      .get('.collapsible-section').eq(0)
```

Comment:
is there a reason we can't add test ids directly to the `<collapsible-section>` nodes? if we did that then this `.get()` statement would be able to tell use which section we're clicking into vs. the 0/1/2 section of content.

Reply:
So I tried getting a hold of collapsible-section using it's heading but looks like it doesn't work in cypress hence I had to do this 0/1/2.

Reply:
hmm that's unfortunate. What you have is still definitely an improvement, though, so let's go with it. Thanks very much for this change!